src/ensembl/src/content/app/entity-viewer/gene-view/components/proteins-list/proteins-list-item-info/ProteinsListItemInfo.tsx

```diff
@@ -153,20 +153,14 @@ const ProteinsListItemInfo = (props: Props) => {
 
   useEffect(() => {
     const abortController = new AbortController();
-    if (
-      summaryStatsLoadingState === LoadingState.LOADING &&
-      !proteinStatsXref
-    ) {
-      // if proteinStatsXref is absent, we cannot fetch relevant data from PDBe; so pretend that we've successfully completed the request
+    if (summaryStatsLoadingState === LoadingState.LOADING && !displayXref) {
+      // if displayXref is absent, we cannot fetch relevant data from PDBe; so pretend that we've successfully completed the request
```

Comment:
Something is wrong in this comment; and maybe also in the logic of this if-statement

Reply:
discussed over slack and zoom during sprint review. Decision is to create a ticket to request for trembl ids and add "Similar proteins" to the protein stats.